.circleci/config.yml

```diff
@@ -122,13 +122,15 @@ commands:
       - run:
           name: "Build Benchmarks - << parameters.benchmark_class >>"
           command: |
-            make benchmarks-basic-build NUM_THREADS=16 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=8
+            make benchmarks-basic-build NUM_THREADS=8 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=8
```

Comment:
Unrelated change?

Reply:
No its related, reducing the instance size to xlarge which has 8 cores.